server/server.go

```diff
@@ -465,6 +473,27 @@ func (s *Server) startMonitoring(secure bool) {
 	}()
 }
 
+func (s *Server) createInternalClient(name string) *client {
+	c := &client{srv: s, nc: nil, opts: defaultOpts, mpay: s.info.MaxPayload, start: time.Now()}
+
+	// Initialize
+	c.typ = CLIENT
+	c.internal = true
```

Comment:
I think this is served by just looking if we have a bw. We could make it a function

```golang
func(c *client) isInternal() bool {
  return c.bw == nil
}
```

Trying to be semi-sensitive to client/connection size.